packages/paste-core/components/breadcrumb/stories/index.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Breadcrumb, BreadcrumbItem} from '../src';
+
+storiesOf('Components|Breadcrumb', module)
+  .addDecorator(withKnobs)
+  .add('Anchors', () => {
+    return (
+      <Breadcrumb>
+        <BreadcrumbItem>
+          <Anchor href="#">Phone Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Anchor href="#">Active Numbers</Anchor>
+        </BreadcrumbItem>
+      </Breadcrumb>
+    );
+  })
+  .add('Anchors and current page ', () => {
+    return (
+      <Breadcrumb>
+        <BreadcrumbItem>
+          <Anchor href="#">Phone Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Anchor href="#">Active Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Text aria-current="page" as="span" fontSize="fontSize20" lineHeight="lineHeight20">
```

Comment:
Do we need this `<Text>` element? Any way around it?

Reply:
Using the following API solves for that:
```
<Breadcrumb>
    <BreadcrumbItem href="#">Phone Numbers</BreadcrumbItem>
    <BreadcrumbItem href="#">Active Numbers</BreadcrumbItem>
    <BreadcrumbItem>Infrrd Germany hotline B</BreadcrumbItem>
</Breadcrumb>
```